tls/s2n_handshake.h

```diff
@@ -180,6 +180,9 @@ struct s2n_handshake {
 
     /* Set to 1 if the RSA verification failed */
     unsigned rsa_failed:1;
+
+    /* Indicates that this is a renegotiation handshake */
+    unsigned renegotiation:1;
```

Comment:
Does this flag mean the same thing as the flag on the connection(conn.secure_renegotiation)?

Reply:
No. "Secure renegotiation" means that the "secure renegotiation" security features / RFC are supported by the peer. That would ideally be true regardless of whether this is an initial or renegotiation handshake. "renegotiation" means that we're actually doing renegotiation.